lib/index.js

```diff
@@ -20,9 +20,11 @@ var AdRoll = module.exports = integration('AdRoll')
   .global('__adroll_loaded')
   .global('adroll_adv_id')
   .global('adroll_custom_data')
+  .global('adroll_email')
   .global('adroll_pix_id')
   .option('advId', '')
   .option('pixId', '')
+  .option('v1', false)
```

Comment:
hmm... the key is that this is literally only to maintain backward compat with their old means of data collection (when event names were legal), whereas now they have IDs that we must map. 

so we *can't* send them event names for new users, but *must* for old users, which means there's no sense in actually exposing an option. it will be hidden and serve more as a "flag" than any sort of opt in change in functionality contract like the salesforce version bump.

that's why i originally called it `legacy`. 

totally open to switching to an option but thought you'd want the context first